app/src/test/java/ingsw/group1/findmyphone/RandomSMSContactGenerator.java

```diff
@@ -0,0 +1,30 @@
+package ingsw.group1.findmyphone;
+
+import java.util.Random;
+
+import ingsw.group1.findmyphone.contacts.SMSContact;
+import ingsw.group1.msglibrary.RandomSMSPeerGenerator;
+
+//TODO specs
```

Comment:
TODO

Reply:
Yeah sry the fixed contact generator is somewhere in part 3 I believe